content/references/http-websocket-apis/public-api-methods/server-info-methods/server_info.md

```diff
@@ -32,21 +33,28 @@ An example of the request format:
     "params": [
         {}
     ]
+},
+{
+    "method": "server_info",
+    "params": [
+        {"counters" : true}
+    ]
 }
 ```
 
 *Commandline*
 
 ```sh
-#Syntax: server_info
+# Syntax: server_info [counters]
+# counters is an optional boolean value, it is used to display performance metrics
 rippled server_info
 ```
 
 <!-- MULTICODE_BLOCK_END -->
 
 [Try it! >](websocket-api-tool.html#server_info)
 
-The request does not take any parameters.
+The request takes in an optional `counters` parameter. It will render metrics, including those that pertain to Job Queue, nodestore and RPC call counters.
```

Comment:
Even if there's only one parameter, we should put it in a table so it's harder to miss. Also, for style reasons, we avoid future tense for situations like this.

```suggestion
The request includes the following parameters:

| Field                 | Type    | Required? | Description |
|:----------------------|:--------|:----------|-------------|
| `counters`            | Boolean | No        | If `true`, return metrics about the job queue, ledger store, and API method activity. The default is `false`. |
```